### PR TITLE
Add blocks to Jinja template 

### PR DESCRIPTION
**Pull request by SegiNyn**

### CODE REVIEW

File: indico/modules/events/registration/templates/management/regform.html [hunk 1]
```diff
@@ -10,109 +10,115 @@
...
+        {% endblock %}
     </div>
 
     <div class="action-box">
-        <div class="section">
-            <div class="icon icon-users"></div>
-            <div class="text">
-                <div class="label">
-                    {% trans %}List of registrations{% endtrans %}
+        {% block form_actions %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/management/regform.html:47]:
```suggestion
        {% block registrant_actions %}
```

File: indico/modules/events/registration/templates/management/regform.html [hunk 2]
```diff
@@ -10,109 +10,115 @@
     {% endif %}
 
     <div class="action-box">
-        <div class="section">
-            <div class="icon icon-settings"></div>
-            <div class="text">
-                <div class="label">
-                    {% trans %}General settings{% endtrans %}
+        {% block form_settings %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/management/regform.html:13]:
```suggestion
        {% block regform_setup %}
```

File: indico/modules/events/registration/templates/display/regform_display.html [hunk 3]
```diff
@@ -94,16 +96,18 @@
...
-            {% call message_box('warning', fixed_width=true, large_icon=true) %}
-                <div class="label">
-                    {% trans %}Participation in this event is moderated{% endtrans %}
-                </div>
-                {% trans -%}
-                    The organizers will have to approve your application.
-                {%- endtrans %}
-            {% endcall %}
-        {% endif %}
+        {% block if_moderated %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/display/regform_display.html:99]:
```suggestion
        {% block moderated_warning %}
```

File: indico/modules/events/registration/templates/display/regform_display.html [hunk 4]
```diff
@@ -71,20 +71,22 @@
...
-                        {%- endif -%}
-                    </div>
-                    <div>
-                        {%- if moderated %}
-                            {%- trans %}In order to apply for this event you have to be logged in.{% endtrans %}
-                        {%- else %}
-                            {%- trans %}In order to register for this event you have to be logged in.{% endtrans %}
-                        {%- endif -%}
-                    </div>
+                    {% block login_required %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/display/regform_display.html:74]:
```suggestion
                    {% block login_required_warning %}
```

File: indico/modules/events/registration/templates/display/conference_home.html [hunk 5]
```diff
@@ -73,14 +75,16 @@
...
                 {% else %}
-                    <a href="{{ url_for('event_registration.display_regform_list', event) }}"
-                       class="i-button next highlight">
-                        {%- if is_form_moderated %}
-                            {%- trans %}Apply now{% endtrans %}
-                        {%- else %}
-                            {%- trans %}Register now{% endtrans %}
-                        {%- endif -%}
-                    </a>
+                    {% block form_link_label %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/display/conference_home.html:78]:
```suggestion
                    {% block register_now_button %}
```

File: indico/modules/events/registration/templates/management/regform_list.html [hunk 6]
```diff
@@ -15,111 +15,119 @@
...
-                            <a href="{{ url_for('.manage_reglist', regform) }}" class="i-button icon-users">
-                                {%- trans %}Registrations{% endtrans -%}
-                                <span class="badge">
-                                    {{- regform.existing_registrations_count -}}
-                                    {% if regform.registration_limit %}
-                                        / {{ regform.registration_limit }}
-                                    {%- endif -%}
-                                </span>
-                            </a>
+        {% block regform_list %}
```
[the remark this answers is not in the document]
Comment (SegiNyn) [reply indico/modules/events/registration/templates/management/regform_list.html:79]: We want to hide this section from some users, with the h3 included.

**Pull request merged**